nh_eobs_mental_health/models/nh_clinical_sql.py

```diff
@@ -367,29 +367,44 @@ def get_refused_wardboard(self, interval):
 
     def get_collect_activities_sql(self, activity_ids_sql):
         sql = self.collect_activities_skeleton.replace(
-            'left join ews1 on ews1.spell_activity_id = spell.id',
-            'left join ews1 on ews1.spell_activity_id = spell.id '
+            'left join ews1 on ews1.spell_activity_id = spell_activity.id',
+            'left join ews1 on ews1.spell_activity_id = spell_activity.id '
+            'LEFT JOIN nh_clinical_spell AS spell '
+            'ON spell.activity_id = spell_activity.id '
+            'LEFT JOIN last_finished_pme AS pme ON pme.spell_id = spell.id '
             'LEFT JOIN refused_last_ews '
-            'ON refused_last_ews.spell_activity_id = spell.id '
+            'ON refused_last_ews.spell_activity_id = spell_activity.id '
+            'AND coalesce(refused_last_ews.date_terminated '
```

Comment:
the coalesce here says if there's no spell.move_date then return True as with the Demo data the move_date on the spell wasn't set, this shouldn't however be an issue on Live as the placement will set the move_date